core/sql/regress/compGeneral/TEST045

```diff
@@ -498,8 +512,36 @@ update statistics for table date_dim on every column;
 update statistics for table date_dim on (d_qoy, d_year);
 update statistics for table store_sales on every column;
 
+
+
+--------------------------------------------------------------------
+?section common_subexpr
```

Comment:
Why did you name this section "common_subexpr"? This regression test is for a feature called common subexpressions, but this section has nothing to do at all with common subexpressions, so that is really misleading.

Reply:
Sorry, didn't realize "common subexpressions" is name of a feature. Will find a better place to put these tests.

Reply:
Yes, I often found it pretty difficult to find a good place to put new tests. Maybe compGeneralTEST015 might work? If you don't find a better place you can also leave the test in TEST045 and we can rename the section to something else, like insert_with_limit_clause.

Reply:
Note the change is already merged, maybe you or I can do this in a future checkin when it's convenient.

Reply:
OK, understood.